api/v1alpha1/warehouse_types.go

```diff
@@ -108,6 +108,12 @@ type WarehouseSpec struct {
 	//
 	// +kubebuilder:validation:MinItems=1
 	Subscriptions []RepoSubscription `json:"subscriptions" protobuf:"bytes,1,rep,name=subscriptions"`
+
+	// FreightCreationCriteria defines criteria that must be satisfied for Freight to be created.
+	// This field has no effect when the FreightCreationPolicy isn't automatic.
```

Comment:
Probably better to say no effect when `Manual` since it seems that `""` is implicitly treated as `Automatic`.

```suggestion
	// FreightCreationCriteria defines criteria that must be satisfied for Freight
	// to be created automatically from new artifacts following discovery. This
	// field has no effect when the FreightCreationPolicy is `Manual`.
```